packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -273,6 +282,19 @@ contract Connext is
     emit AssetRemoved(canonicalId, msg.sender);
   }
 
+  /**
+   * @notice Used to add approved relayer
+   * @param relayer - The relayer address to add or remove
+   * @param approved - approved or not
+   */
+  function setupRelayer(address relayer, bool approved) external override onlyOwner {
+    if (relayer == address(0)) revert Connext__setupRelayer_badAddress();
+    if (approvedRelayers[relayer] == approved) revert Connext__setupRelayer_alreadyDid();
+    approvedRelayers[relayer] = approved;
+
+    emit RelayerSetup(relayer, approved);
+  }
+
```

Comment:
I'm inclined to have separate functions for adding and removing relayers (same as we do with assets/routers)

Reply:
Trying to reduce contract bytecode size as much as I can. lol

Fixed.
[https://github.com/connext/nxtp/commit/ba57917bf911ee3be020d48634f3518ffe8a8750](url)